hc-sr04-range-sensor/Makefile

```diff
@@ -61,6 +61,14 @@ OUT := out
 ELF0 := $(OUT)/pru-halt.elf
 ELF1 := $(OUT)/hc-sr04-range-sensor.elf
 
+#Installation, start stop related variable
+PRU0        :=/sys/class/remoteproc/remoteproc1
+PRU1        :=/sys/class/remoteproc/remoteproc2
+STATESRUNNING   :=running
+STATESSTOPED   :=offline
+CURRENTSTATE0:=$(shell cat $(PRU0)/state)
```

Comment:
This will break building firmware on a PC host.

I don't want sysfs to be poked when I'm building firmware.